src/utils/tools.js

```diff
@@ -22,36 +22,63 @@ class SqlUtil {
 
 //Useful tool for validating user inputs
 class inputValidation {
+  
+/**
+   * Checks content type
+   * @param {req} req response
+   * @param {res} res User input parameter
+   */
+  static checkContentType(req, res){
+    var contype = req.headers['content-type'];
+    if (!contype || contype.indexOf('application/json') !== 0) {
+      res.sendStatus(415);
+      return false;
+    }
+    else return true;
+  }
+
   /**
    * Checks if json contains null value or not
+   * @param {res} res response
    * @param {JSON} json User input parameter
    * @param {Array} exceptions Nullable exceptions.
    * @return {boolean} If Json includes null, return true.
    */
-  static includesNullorEmpty(json, exceptions = []) {
+  static checkNullorEmpty(res, json, exceptions = []) {
```

Comment:
- changed the function name
- return status code and error message

Reply:
One potential problem I see with bring in the error handling here, is that these functions will only ever be able to be used on the controllers. Is there any situation where they might be useful in a model?

These changes definitely make it more convenient. I am just worried about modularity. What do you think?

Reply:
Yeah I agree, it would be nice to have these functions available (other than controllers). 
I think I figured out the proper way and I changed it some. Reference: https://scoutapm.com/blog/express-error-handling